index.html

```diff
@@ -0,0 +1,347 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="assets/styles/style.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+    <section class="showcase">
+        <header class="main-header">
+            <nav class="">
+                <a class="logo" href="/">
+                    <h1>Current Logo</h1>
+                </a>
+                <div class="">
+                    <i class='bx bx-menu'></i>
+                </div>
+            </nav>
+        </header>
+
+        <div class="showcase-description">
+            <h1>
+                I’m Djibril. <br>
+                Glad to see you!
+            </h1>
+
```

Comment:
- [ ] Good design but you still need to improve on little things for your project to match the Figma design, kindly make your `h1` tag look the same as Figma one, that see you shouldn't be on the same line, and fix well your background-image for your name to be clean and readable :wink: 
Your Design :point_down: 
![Screenshot from 2023-04-28 15-50-13](https://user-images.githubusercontent.com/88894525/235169630-066438a6-fa5b-4781-81c8-530ccf061398.png)

Figma design 
![Screenshot from 2023-04-28 15-56-36](https://user-images.githubusercontent.com/88894525/235169919-1ff79729-b0eb-409f-ae7a-9d5826993d4e.png)